node/middlewares/prepare.ts

```diff
@@ -10,7 +10,7 @@ export function prepare (explicitlyAuthenticated: boolean) {
       const isImpersonated = !!sessionPayload?.sessionData?.namespaces?.impersonate?.storeUserId?.value
       const vtexIdClientCookieName = isImpersonated ? 'VtexIdclientAutCookie' : `VtexIdclientAutCookie_${account}`
       VtexIdclientAutCookie = sessionPayload?.sessionData?.namespaces?.cookie?.[vtexIdClientCookieName]?.value
-      if (!explicitlyAuthenticated) {
+      if (!explicitlyAuthenticated && Math.floor(Math.random() * 100) === 0) {
```

Comment:
what if we extract to a const or add a comment to explain why we are doing that?